client/blocks/checklist/checklist-task.jsx

```diff
@@ -56,8 +56,9 @@ export class ChecklistTask extends PureComponent {
 			duration,
 			title,
 			translate,
+			buttonText,
 		} = this.props;
-		const { buttonText = translate( 'Do it!' ) } = this.props;
+		//{ buttonText = translate( 'Do it!' ) } = this.props;
```

Comment:
Could you please remove this commented line?

Reply:
Awesome! I will remove the commented line but I do have a question about ```translate()```.  This text probably still needs to be translated, so would I also need to fit ```translate()``` in there somewhere? I was reading the i18n and it mentioned using args in order to use variables.

Reply:
Ideally, all these should be wrapped in a `translate` call, yes.

You can add that function as an **external** dependency like so:
`import { translate } from 'i18n-calypso';`

It does make it a more complex from a design perspective because different languages will have different string lengths.

I'd look into:
* Specifying reasonable `max-width` values at appropriate breakpoints somewhere in [this file](https://github.com/Automattic/wp-calypso/blob/3ea0e1f44871d84f18a654a9a4f81ad88bb174fa/client/blocks/checklist/style.scss)
* Adding the `buttonText` as the value of a `title` attribute on the `Button` component [here](https://github.com/Automattic/wp-calypso/blob/3ea0e1f44871d84f18a654a9a4f81ad88bb174fa/client/blocks/checklist/checklist-task.jsx#L85) so that strings that run long are still accessible via mouse-over